Source/WebKit/UIProcess/BackingStore.h

```diff
@@ -25,10 +25,11 @@
 
 #pragma once
 
-#if USE(GRAPHICS_LAYER_WC) || (USE(CAIRO) && !PLATFORM(WPE))
+#if USE(GRAPHICS_LAYER_WC) || PLATFORM(GTK) || (USE(CAIRO) && !PLATFORM(WPE))
 
 #include <WebCore/IntSize.h>
 #include <WebCore/PlatformImage.h>
+#include <WebCore/RefPtrCairo.h>
```

Comment:
This will break some of the porting I'm doing for other platform. Please move it to the `USE(CAIRO) || PLATFORM(GTK)` block.